config/routes.rb

```diff
@@ -62,6 +62,8 @@
   match '/edit/email' => 'users/edit_email#update', via: [:patch, :put]
   get '/edit/phone' => 'users/edit_phone#edit'
   match '/edit/phone' => 'users/edit_phone#update', via: [:patch, :put]
+  get '/settings/password' => 'users/edit_password#edit'
+  patch '/settings/password' => 'users/edit_password#update'
```

Comment:
Why not follow the convention above and use  `/edit/password`.  We should at least be consistent.